app/services/room_collection_service.py

```
@@ -566,96 +575,44 @@ def _extract_representative_keywords(cls, keywords: List[str]) -> List[str]:
-                    text = value.strip().lower()
-                    if text:
-                        fragments.append(text)
-        # 동일 텍스트는 한 번만 남겨 키워드 판별 근거를 단순화한다.
-        return " ".join(dict.fromkeys(fragments))
-
     @classmethod
     def _evaluate_rehearsal_domain(
         cls,
         source_hint: Optional[Dict[str, Any]],
         business: Dict[str, Any],
         rooms: List[Dict[str, Any]],
```

Ruff (0.15.4)

[warning] Unused class method argument: `rooms`

(ARG003)